src/tags/object/Image.js

```diff
@@ -289,11 +291,20 @@ const Model = types.model({
   },
 
   get parsedValue() {
-    return parseValue(self.value, self.store.task.dataObj);
+    return self._parsedValue = self._parsedValue ?? parseValue(self.value, self.store.task.dataObj);
```

Comment:
why do you need this temporary private vars here and in `parsedValueList`?

Reply:
those are not temp, it's cache

Reply:
but that's the getter, so it should not be called randomly. and if `value` is changed for some reason (LSF re-init from DM for example) it should be recalculated.

Reply:
yeah, you're right